File: dev-superior-algoritmo-e-logica-de-programacao/cpp/exercicios/troco.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double precoUnitario, dinheiroRecebido, troco;
    int qtd;

    cout << "Preco unitario do produto: ";
    cin >> precoUnitario;

    cout << "Quantidade comprada: ";
    cin >> qtd;

    cout << "Dinheiro recebido: ";
    cin >> dinheiroRecebido;

    troco = dinheiroRecebido - (precoUnitario * qtd);

    cout << fixed << setprecision(2);
    cout << "TROCO = " << troco << endl;

	return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/troco.py
precoUnitario: float; dinheiroRecebido: float; troco: float
qtd: int

precoUnitario = float(input("Preco unitario do produto: "))
qtd = int(input("Quantidade comprada: "))
dinheiroRecebido = float(input("Dinheiro recebido: "))

troco = dinheiroRecebido - (precoUnitario * qtd)

print(f"TROCO = {troco:.2f}")